frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/common/useEditFieldForm.ts

```diff
@@ -90,6 +94,9 @@ export const useEditFieldForm = <FormShape, FieldShape extends FormField>({
 
   const { editFieldMutation } = useEditFormField()
   const { createFieldMutation } = useCreateFormField()
+  const [, setisAdminFeedbackEligible] = useSessionStorage<boolean>(
```

Comment:
nit:
```suggestion
  const [, setIsAdminFeedbackEligible] = useSessionStorage<boolean>(
```

Reply:
Thank you! [8262cd3](https://github.com/opengovsg/FormSG/pull/6465/commits/8262cd3134eb8336dae6b157275e98053e87ca79)